python/pyspark/mllib/clustering.py

```diff
@@ -140,7 +140,7 @@ class BisectingKMeans(object):
 
     @classmethod
     @since('2.0.0')
-    def train(self, rdd, k=4, maxIterations=20, minDivisibleClusterSize=1.0, seed=-1888008604):
+    def train(self, rdd, k=4, maxIterations=20, minDivisibleClusterSize=1.0, seed=None):
```

Comment:
This was the only non-test instance I could see where Pyspark sets a seed to anything but None.